pkg/controller/image-stream-tag-reconciler/reconciler.go

```diff
@@ -246,9 +279,13 @@ func (r *reconciler) getPublishJob(br *branchReference, ciOPConfig *cioperatorap
 }
 
 func (r *reconciler) isJobRunningForCommit(job *prowconfig.Postsubmit, commitSHA string, istRef *branchReference) (bool, error) {
+	jobNameLabelValue := job.Name
+	// Label values are capped at 63 characters and the job name frequently exceeds that.
+	if len(jobNameLabelValue) > 63 {
+		jobNameLabelValue = job.Name[0:62]
+	}
 	labelSelector := ctrlruntimeclient.MatchingLabels{
-		// Label values are capped at 63 characters and the job name frequently exceeds that.
-		kube.ProwJobAnnotation: job.Name[0:62],
+		kube.ProwJobAnnotation: jobNameLabelValue,
```

Comment:
do we seriously call this `annotation` :facepalm:

Reply:
To be fair, its both an annotation and a label